internal/service/alarms/internal/dictionary/suite_test.go

```diff
@@ -0,0 +1,13 @@
+package dictionary
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestConfig(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Dictionary Suite")
+}
```

Comment:
Ultimately there will be only one suite (alarms_suite?) file all our tests? Sorry about the dumb q but forgot how ginkgo recommends structuring projects

Reply:
Yes, at some point, I was thinking about running a simulated api server with envtest, that's why I put it here. Will change it.

Reply:
Best practice is to have a test suite per package. I left it unchanged.